### ISSUE TITLE
fail with a descriptive error when a specified config file is not readable

### ISSUE THREAD

**Issue by derEremit** [dominictarr/rc#75]
I don't know if this is against the concept of rc, but when I specify a config file with --config I would like to see an error when I just typed the config path wrong for example.

the default behavior could produce hours of false debug sessions because I don't realize the config is just not loaded!

**Comment by dominictarr** [comment]
yes, I think this is reasonable. although I can also imagine that someone may depend on this behavior so I'd mark this a breaking change. I would merge this pull request!
